src/ROUTES.js

```diff
@@ -53,6 +52,7 @@ export default {
     SETTINGS_CONTACT_METHODS,
     SETTINGS_CONTACT_METHOD_DETAILS: `${SETTINGS_CONTACT_METHODS}/:contactMethod/details`,
     getEditContactMethodRoute: contactMethod => `${SETTINGS_CONTACT_METHODS}/${encodeURIComponent(contactMethod)}/details`,
+    SETTINGS_NEW_CONTACT_METHOD: 'settings/profile/contact-methods/new',
```

Comment:
To keep this consistent with above routes

```suggestion
    SETTINGS_NEW_CONTACT_METHOD: `${SETTINGS_CONTACT_METHODS}/new`,
```